docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -99,16 +99,20 @@ stored on the data node according to the type of data node:
 
 It is also possible to partially read the contents of data nodes, which comes in handy when dealing with large amounts
 of data.
-This can be achieved using the `DataNode.filter()^` method:
+This can be achieved by providing an operator, a Tuple of (field_name, value, comparison operator),
```

Comment:
I suspect that [field_name, value, comparison operator]  should be in italics...